Debug Prompt Logic Change With `$Host`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The Debug Prompt isn't consistently firing. I think we should use `$Host.Runspace.Debugger.InBreakpoint` instead of `$PSDebugContext` and `Get-PSCallStack`. Or if we could just add it in as `-or` logic?

https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/src/shell/scripts/omp.ps1#L362

Here is a repo of it not working based on the code above.

```pwsh
new-module foo {
    set-item function:\prompt { set-pr }
    function set-pr () { if( $null -ne $PSDebugContext ) { 'dbg> ' } else { 'nondbg> ' } }
    Export-ModuleMember -Function prompt
} | Import-Module -Global
& { wait-debugger; sleep 3 }
```

If you change the logic to `if ( $Host.Runspace.Debugger.InBreakpoint )` it works.

#### Reference
https://github.com/PowerShell/vscode-powershell/issues/3980

### Theme
```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#21c7a8",
          "foreground": "#011627",
          "leading_diamond": "\u256d\u2500\ue0b6",
          "properties": {
            "windows": "\ue62a"
          },
          "style": "diamond",
          "template": " {{ if .WSL }}WSL at {{ end }}{{.Icon}}  ",
          "trailing_diamond": "\ue0b0",
          "type": "os"
        },
        {
          "background": "#ef5350",
          "foreground": "#ffeb95",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "template": " \uf292 ",
          "type": "root"
        },
        {
          "background": "#82AAFF",
          "foreground": "#011627",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "folder_icon": "\uf07c ",
            "folder_separator_icon": "<#011627>\ue0b1</> ",
            "home_icon": " \ueb06 ",
            "style": "agnoster"
          },
          "style": "powerline",
          "template": "{{ path .Path .Location }}",
          "type": "path"
        },
        {
          "background": "#addb67",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#e4cf6a{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#f78c6c{{ end }}",
            "{{ if gt .Ahead 0 }}#C792EA{{ end }}",
            "{{ if gt .Behind 0 }}#c792ea{{ end }}"
          ],
          "foreground": "#011627",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "branch_icon": "\ue725 ",
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_upstream_icon": true,
            "fetch_worktree_count": true
          },
          "style": "powerline",
          "template": " {{ url .UpstreamIcon .UpstreamURL }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }} ",
          "type": "git"
        },
        {
          "background": "#575656",
          "foreground": "#d6deeb",
          "leading_diamond": "<transparent,#575656>\ue0b0</>",
          "properties": {
            "style": "roundrock",
            "threshold": 0
          },
          "style": "diamond",
          "template": " {{ .FormattedMs }}",
          "trailing_diamond": "\ue0b4",
          "type": "executiontime"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "right",
      "overflow": "break",
      "segments": [
        {
          "background": "#d6deeb",
          "foreground": "#011627",
          "leading_diamond": "\ue0b6",
          "style": "diamond",
          "template": "\uf489  {{ .Name }} ",
          "trailing_diamond": "<transparent,#d6deeb>\ue0b2</>",
          "type": "shell"
        },
        {
          "background": "#8f43f3",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue266 {{ round .PhysicalPercentUsed .Precision }}% ",
          "trailing_diamond": "<transparent,#8f43f3>\ue0b2</>",
          "type": "sysinfo"
        },
        {
          "background": "#ffffff",
          "foreground": "#ce092f",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue753 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "angular"
        },
        {
          "background": "#565656",
          "foreground": "#faa029",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} ",
          "trailing_diamond": "<transparent,#565656>\ue0b2</>",
          "type": "aws"
        },
        {
          "background": "#ffffff",
          "foreground": "#337bb6",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uebd8 {{ .EnvironmentName }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "az"
        },
        {
          "background": "#ffffff",
          "foreground": "#5398c2",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uf104<#f5bf45>\uf0e7</>\uf105 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "azfunc"
        },
        {
          "background": "#5a7a94",
          "foreground": "#100e23",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue311  cds {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#5a7a94>\ue0b2</>",
          "type": "cds"
        },
        {
          "background": "#ffffff",
          "foreground": "#000000",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uE370 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#000000>\ue0b2</>",
          "type": "crystal"
        },
        {
          "background": "#ffffff",
          "foreground": "#1d5185",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uf40a  cf {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "cf"
        },
        {
          "background": "#ffffff",
          "foreground": "#1d5185",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uf40a {{if .Org }}{{ .Org }}{{ end }}{{ if .Space }}/{{ .Space }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "cftarget"
        },
        {
          "background": "#d2d2d2",
          "foreground": "#01a300",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "<#2829b2>\ue61e</> <#be1818>\ue61d</>  cmake {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#d2d2d2>\ue0b2</>",
          "type": "cmake"
        },
        {
          "background": "#e1e8e9",
          "foreground": "#055b9c",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue798 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#e1e8e9>\ue0b2</>",
          "type": "dart"
        },
        {
          "background": "#0e0e0e",
          "foreground": "#0d6da8",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue77f  {{ if .Unsupported }}\uf071{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#0e0e0e>\ue0b2</>",
          "type": "dotnet"
        },
        {
          "background": "#06A4CE",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue28e {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#06A4CE>\ue0b2</>",
          "type": "flutter"
        },
        {
          "background": "#ffffff",
          "foreground": "#06aad5",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue626 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "go"
        },
        {
          "background": "#100e23",
          "foreground": "#906cff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue61f {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#100e23>\ue0b2</>",
          "type": "haskell"
        },
        {
          "background": "#ffffff",
          "foreground": "#ec2729",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue738 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "java"
        },
        {
          "background": "#945bb3",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "<#ca3c34>\ue624</> {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#945bb3>\ue0b2</>",
          "type": "julia"
        },
        {
          "background": "#906cff",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "K {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#906cff>\ue0b2</>",
          "type": "kotlin"
        },
        {
          "background": "#316ce4",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\uf308 {{.Context}} :: {{if .Namespace}}{{.Namespace}}{{else}}default{{end}} ",
          "trailing_diamond": "<transparent,#316ce4>\ue0b2</>",
          "type": "kubectl"
        },
        {
          "background": "#ffffff",
          "foreground": "#000081",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue620 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "lua"
        },
        {
          "background": "#303030",
          "foreground": "#3C873A",
          "leading_diamond": "\ue0b2",
          "properties": {
            "fetch_package_manager": true,
            "npm_icon": "<#cc3a3a>\ue71e</> ",
            "yarn_icon": "<#348cba>\ue6a7</> "
          },
          "style": "diamond",
          "template": "\ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} ",
          "trailing_diamond": "<transparent,#303030>\ue0b2</>",
          "type": "node"
        },
        {
          "background": "#6488c0",
          "foreground": "#1e293b",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "Nx {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#6488c0>\ue0b2</>",
          "type": "nx"
        },
        {
          "background": "#41436d",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue769 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#41436d>\ue0b2</>",
          "type": "perl"
        },
        {
          "background": "#787CB5",
          "foreground": "#000000",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue73d {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#787CB5>\ue0b2</>",
          "type": "php"
        },
        {
          "background": "#306998",
          "foreground": "#FFE873",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue235  {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }}",
          "trailing_diamond": "<transparent,#306998>\ue0b2</>",
          "type": "python"
        },
        {
          "background": "#b9bbbf",
          "foreground": "#1c68bd",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "R {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#b9bbbf>\ue0b2</>",
          "type": "r"
        },
        {
          "background": "#ffffff",
          "foreground": "#9c1006",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue791 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "ruby"
        },
        {
          "background": "#ffffff",
          "foreground": "#000000",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue7a8 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#ffffff>\ue0b2</>",
          "type": "rust"
        },
        {
          "background": "#fe562e",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": "\ue755 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "trailing_diamond": "<transparent,#fe562e>\ue0b2</>",
          "type": "swift"
        },
        {
          "background": "#234d70",
          "foreground": "#d6deeb",
          "leading_diamond": "\ue0b2",
          "properties": {
            "time_format": "15:04:05"
          },
          "style": "diamond",
          "template": "\ue641 {{ .CurrentDate | date .Format }}",
          "trailing_diamond": "\ue0b4",
          "type": "time"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "#21c7a8",
          "style": "plain",
          "template": "\u2570\u2500",
          "type": "text"
        },
        {
          "background": "#1DB954",
          "foreground": "#011627",
          "leading_diamond": "\ue0b6",
          "properties": {
            "playing_icon": "\uf1bc "
          },
          "style": "diamond",
          "template": " {{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} ~ {{ .Track }}{{ end }} ",
          "trailing_diamond": "\ue0c6 ",
          "type": "spotify"
        },
        {
          "foreground": "#22da6e",
          "foreground_templates": ["{{ if gt .Code 0 }}#ef5350{{ end }}"],
          "properties": {
            "always_enabled": true
          },
          "style": "plain",
          "template": "\ue285\ue285",
          "type": "status"
        }
      ],
      "type": "prompt"
    }
  ],
  "console_title_template": "{{ .Folder }}",
  "final_space": true,
  "secondary_prompt": {
    "background": "transparent",
    "foreground": "#046e29",
    "template": "\uf48b\uf460 " 
  },
  "debug_prompt": {
    "background": "transparent",
    "foreground": "#ffffff",
    "template": "<#FFA500,transparent>\uf188</>\u2800<#046e29,transparent>\uF054</> "
  },
  "version": 2
}
```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 21.16.0

Shell: oh-my-posh (7.4.3)

Prompt:

╭─   ]8;;file:C:/Users/user\  ]8;;\ 0ms                                                                                                                                                                                                                                   oh-my-posh  92%  06:55:28
╰─ 

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Path(true)                                 -   1 ms
Executiontime(true)                        -   0 ms
Shell(true)                                -   0 ms
Sysinfo(true)                              -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Status(true)                               -   0 ms

Run duration: 206.3172ms

Cache path: C:\Users\user\AppData\Local\oh-my-posh

Config path: C:\Users\user\OneDrive\Documents\oh-my-posh\themes\night-owl_mac.omp.json

Logs:

[DEBUG] 06:55:28.015 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.016 shell.go:Getenv(TERM_PROGRAM) - 590.5µs
[DEBUG] 06:55:28.016 shell.go:Getenv:326 → C:\Users\user\AppData\Local
[TRACE] 06:55:28.016 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 06:55:28.016 shell_windows.go:CachePath() - 543.6µs
[DEBUG] 06:55:28.017 shell.go:Getenv:326 → C:\Users\user\OneDrive\Documents\oh-my-posh\themes\night-owl_mac.omp.json
[TRACE] 06:55:28.017 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 06:55:28.017 shell.go:Shell:601 → no shell name provided in flags, trying to detect it
[DEBUG] 06:55:28.035 shell.go:Shell:609 → process name: oh-my-posh.exe
[TRACE] 06:55:28.035 shell.go:Shell() - 18.2971ms
[TRACE] 06:55:28.035 shell.go:resolveConfigPath() - 18.2971ms
[TRACE] 06:55:28.035 shell.go:Init() - 20.0597ms
[TRACE] 06:55:28.035 shell.go:Flags() - 0s
[TRACE] 06:55:28.039 config.go:loadConfig() - 3.9331ms
[TRACE] 06:55:28.039 shell.go:Flags() - 0s
[DEBUG] 06:55:28.039 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.039 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 06:55:28.039 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[TRACE] 06:55:28.039 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.039 shell_windows.go:WindowsRegistryKeyValue:217 → ColorizationColor(DWORD): 0xC44C4A48
[TRACE] 06:55:28.039 shell.go:Shell() - 0s
[DEBUG] 06:55:28.039 shell.go:Getenv:326 → 7.4.3
[TRACE] 06:55:28.039 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 06:55:28.039 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 06:55:28.039 text.go:Render:72 → Rendering template: {{ .Folder }}
[TRACE] 06:55:28.040 shell_windows.go:Root() - 192.5µs
[TRACE] 06:55:28.040 shell.go:Shell() - 0s
[TRACE] 06:55:28.040 shell.go:StatusCodes() - 0s
[TRACE] 06:55:28.040 shell_windows.go:IsWsl() - 0s
[DEBUG] 06:55:28.040 shell.go:TemplateCache:782 → environment: [ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\user\AppData\Roaming BAT_PAGER=less --raw-control-chars --ignore-case --quit-if-one-screen --quiet BAT_THEME=Visual Studio Dark+ ChocolateyInstall=C:\ProgramData\chocolatey ChocolateyToolsLocation=C:\tools CLASS_EXPLORER_TRUE_CHARACTER=✓ CLINK_DIR=C:\Program Files (x86)\clink CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPlus_TieredCompilation=0 COMPUTERNAME=comp ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DOTNET_TieredCompilation=0 DriverData=C:\Windows\System32\Drivers\DriverData ECW_PLUGIN_PATH=C:\Program Files (x86)\Common Files\eClinicalWorks\plugin\WinProjectE.exe  EDITOR=code FZF_ALT_C_COMMAND=fd.exe --type directory FZF_DEFAULT_COMMAND=fd.exe FZF_DEFAULT_OPTS=--color='hl+:#5fd7ff,hl:#5f87af,bg:-1,bg+:-1,fg:-1,fg+:#919092' --color='prompt:#00897b,pointer:#c386f1,marker:#ff479c' --prompt=' ' --pointer='ﰉ ' --marker='' --border --padding=1 --margin=1 --history='C:\Users\user\OneDrive\Documents\.fzf_history' --bind change:first GIT_EDITOR=vim HOMEDRIVE=C: HOMEPATH=\Users\user LESS=--raw-control-chars --ignore-case --quit-if-one-screen --quiet LESSCHARSET=utf-8 LESSEDIT=%E ?l--goto %g\:%l:%g. LESSHISTFILE=C:\Users\user/_lesshst LOCALAPPDATA=C:\Users\user\AppData\Local LOGONSERVER=\\vendDC1 LUA_DEV=C:\Program Files (x86)\Lua\5.1 LUA_PATH=;;C:\Program Files (x86)\Lua\5.1\lua\?.luac NUMBER_OF_PROCESSORS=8 OneDrive=C:\Users\user\OneDrive OneDriveCommercial=C:\Users\user\OneDrive OS=Windows_NT PAGER=less Path=C:\Program Files\PowerShell\7;C:\Program Files (x86)\Common Files\Oracle\Java\java8path;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Sennheiser\SoftphoneSDK\;C:\Program Files (x86)\Microsoft SQL Server\110\Tools\Binn\ManagementStudio\;C:\Program Files (x86)\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files (x86)\Microsoft Visual Studio 11.0\Common7\IDE\PrivateAssemblies\;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\dotnet\;C:\Program Files (x86)\Microsoft Team Foundation Server 2012 Power Tools\;C:\ProgramData\chocolatey\bin;C:\Program Files\Go\bin;C:\Program Files (x86)\dotnet-core-uninstall\;C:\tools\gsudo\Current;C:\Program Files (x86)\Lua\5.1;C:\Program Files (x86)\Lua\5.1\clibs;C:\tools\pdftk-java;C:\Program Files (x86)\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files\Git\cmd;C:\Program Files\OpenSSH-Win64;C:\ProgramData\chocolatey\lib\gh\tools\;C:\Program Files\PowerShell\7\;C:\Users\user\scoop\shims;C:\Users\user\AppData\Local\Microsoft\WindowsApps;;C:\Users\user\AppData\Local\Programs\oh-my-posh\bin;C:\Users\user\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\user\.dotnet\tools;C:\Users\user\OneDrive\Documents\exe;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64 PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.wlua;.lexe;.CPL POSH_AZURE_ENABLED=False POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=71 POSH_GIT_ENABLED=False POSH_INSTALLER=winget POSH_PID=34392 POSH_SHELL_VERSION=7.4.3 POSH_THEME=C:\Users\user\OneDrive\Documents\oh-my-posh\themes\night-owl_mac.omp.json POSH_THEMES_PATH=C:\Users\user\scoop\apps\oh-my-posh\current\themes POWERLINE_COMMAND=oh-my-posh POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Enterprise PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 142 Stepping 12, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=8e0c ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\user\OneDrive\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public RIPGREP_CONFIG_PATH=C:\Users\user\OneDrive\Documents\.ripgreprc SESSIONNAME=Console SHELL=cmd.exe SMS_ADMIN_UI_PATH=C:\program files (x86)\ConfigMgr\bin\i386 SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\user\AppData\Local\Temp TFSPowerToolDir=C:\Program Files (x86)\Microsoft Team Foundation Server 2012 Power Tools\ TMP=C:\Users\user\AppData\Local\Temp UATDATA=C:\WINDOWS\CCM\UATData\D9F8C395-CAB8-491d-B8AC-179A1FE1BE77 USERDNSDOMAIN=domain.NET USERDOMAIN=doma USERDOMAIN_ROAMINGPROFILE=doma USERNAME=user USERPROFILE=C:\Users\user VISUAL=code windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=b0b284bc-d3b4-4eac-84dd-f3d6ba10b18e ZES_ENABLE_SYSMAN=1]
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.040 shell.go:Pwd:356 → C:\Users\user
[TRACE] 06:55:28.040 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.040 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.040 shell_windows.go:IsWsl() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.040 shell.go:User:494 → user
[TRACE] 06:55:28.040 shell.go:User() - 0s
[DEBUG] 06:55:28.040 shell.go:Host:511 → comp
[TRACE] 06:55:28.040 shell.go:Host() - 0s
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.040 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.040 shell.go:Getenv(SHLVL) - 0s
[TRACE] 06:55:28.040 shell.go:TemplateCache() - 269.5µs
[DEBUG] 06:55:28.040 shell.go:Getenv:326 → 71
[TRACE] 06:55:28.040 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 06:55:28.040 shell.go:Getenv:326 → 1
[TRACE] 06:55:28.040 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 06:55:28.040 shell.go:Flags() - 0s
[TRACE] 06:55:28.040 shell.go:Flags() - 0s
[TRACE] 06:55:28.040 shell.go:Shell() - 0s
[TRACE] 06:55:28.040 shell.go:Shell() - 0s
[TRACE] 06:55:28.040 shell.go:Flags() - 0s
[TRACE] 06:55:28.040 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.040 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.040 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.040 segment.go:SetEnabled:560 → Segment: Executiontime
[TRACE] 06:55:28.040 shell.go:Flags() - 0s
[TRACE] 06:55:28.040 shell.go:Flags() - 0s
[DEBUG] 06:55:28.041 properties.go:GetBool:22 → always_enabled: false
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[TRACE] 06:55:28.041 shell.go:ExecutionTime() - 0s
[TRACE] 06:55:28.041 shell.go:Flags() - 0s
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.041 properties.go:GetFloat64:34 → threshold: 0.000000
[TRACE] 06:55:28.041 shell.go:Flags() - 0s
[DEBUG] 06:55:28.041 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.041 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.041 properties.go:GetString:28 → roundrock
[DEBUG] 06:55:28.041 segment.go:SetEnabled:560 → Segment: Os
[DEBUG] 06:55:28.041 segment.go:SetEnabled:560 → Segment: Root
[TRACE] 06:55:28.041 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.041 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.041 segment.go:SetEnabled:560 → Segment: Path
[DEBUG] 06:55:28.041 properties.go:GetString:28 → 
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[TRACE] 06:55:28.041 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:Shell() - 0s
[TRACE] 06:55:28.041 shell.go:Shell() - 0s
[DEBUG] 06:55:28.041 shell.go:Getenv:326 → C:\Users\user\AppData\Local
[TRACE] 06:55:28.041 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 06:55:28.041 shell_windows.go:Root() - 0s
[DEBUG] 06:55:28.041 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 06:55:28.041 properties.go:GetString:28 → 
[DEBUG] 06:55:28.041 properties.go:GetString:28 → 
[DEBUG] 06:55:28.041 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.041 properties.go:GetString:28 →   
[DEBUG] 06:55:28.041 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[TRACE] 06:55:28.041 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.041 properties.go:GetString:28 → NO DATA
[TRACE] 06:55:28.041 shell.go:Pwd() - 0s
[TRACE] 06:55:28.041 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.041 shell.go:StackCount() - 0s
[DEBUG] 06:55:28.042 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 06:55:28.042 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 06:55:28.042 win32_windows.go:isWriteable:303 → current user is member of S-1-5-21-58594938-119884983-774919444-113688
[DEBUG] 06:55:28.042 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 06:55:28.042 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 06:55:28.042 shell_windows.go:DirIsWritable() - 1.1252ms
[TRACE] 06:55:28.042 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.059 shell.go:CommandPath:553 → C:\Program Files\Git\cmd\git.exe
[TRACE] 06:55:28.059 shell.go:CommandPath(git.exe) - 18.8087ms
[TRACE] 06:55:28.059 shell.go:HasCommand(git.exe) - 18.8087ms
[TRACE] 06:55:28.059 shell.go:Pwd() - 0s
[ERROR] 06:55:28.060 shell.go:HasParentFilePath:697 → CreateFile .git: The system cannot find the file specified.
[TRACE] 06:55:28.060 shell.go:HasParentFilePath(.git) - 592.3µs
[DEBUG] 06:55:28.060 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 06:55:28.060 text.go:Render:72 → Rendering template:  {{ .FormattedMs }}
[DEBUG] 06:55:28.060 text.go:Render:72 → Rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}}  
[DEBUG] 06:55:28.060 text.go:Render:72 → Rendering template: {{ path .Path .Location }}
[TRACE] 06:55:28.061 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.061 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.062 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.062 text.go:Render:72 → Rendering template: powerline
[DEBUG] 06:55:28.062 text.go:Render:72 → Rendering template: powerline
[DEBUG] 06:55:28.062 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.062 text.go:Render:72 → Rendering template: powerline
[DEBUG] 06:55:28.062 text.go:Render:72 → Rendering template: diamond
[TRACE] 06:55:28.062 shell.go:Shell() - 0s
[TRACE] 06:55:28.062 shell.go:Shell() - 0s
[TRACE] 06:55:28.062 shell.go:Shell() - 0s
[TRACE] 06:55:28.062 shell.go:Shell() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Aws
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → display_default: true
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_VAULT) - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_DEFAULT_PROFILE) - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_PROFILE) - 0s
[DEBUG] 06:55:28.062 properties.go:GetInt:40 → precision: 2
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_REGION) - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Time
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[DEBUG] 06:55:28.062 properties.go:GetString:28 → 15:04:05
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_DEFAULT_REGION) - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(AWS_CONFIG_FILE) - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Getenv(JAVA_HOME) - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Angular
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Shell
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [angular.json]
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Shell() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Julia
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Java
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.jl]
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [pom.xml build.gradle.kts build.sbt .java-version .deps.edn project.clj build.boot *.java *.class *.gradle *.jar *.clj *.cljc]
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[ERROR] 06:55:28.062 shell.go:FileContent:464 → open C:\Users\user/.aws/config: The system cannot find the path specified.
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:FileContent(C:\Users\user/.aws/config) - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Php
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Flutter
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.php composer.json composer.lock .php-version blade.php]
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: [.dart_tool]
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Cftarget
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Ruby
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → C:\Users\user\AppData\Local
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Perl
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Rust
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Cmake
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.rs Cargo.toml Cargo.lock]
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Go
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.cmake CMakeLists.txt]
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[DEBUG] 06:55:28.062 shell.go:Pwd() - Bool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetString:28 → never
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Dart
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Swift
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.swift *.SWIFT]
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Haskell
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.hs *.lhs stack.yaml package.yaml *.cabal cabal.project]
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 properties.go:GetString:28 → environment
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: [.dart_tool]
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [.perl-version *.pl *.pm *.t]
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Python
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 properties.go:GetString:28 → context

[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: R
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.R *.Rmd *.Rsx *.Rda *.Rd *.Rproj .Rproj.user]
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Cds
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Sysinfo
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Az
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [.cdsrc.json .cdsrc-private.json *.cds]
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetString:28 → first_match
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[DEBUG] 06:55:28.062 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[TRACE] 06:55:28.062 shell.go:Getenv(AZURE_CONFIG_DIR) - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 properties.go:GetString:28 → lua
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:Flags() - 0s
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[TRACE] 06:55:28.062 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Lua
[DEBUG] 06:55:28.062 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.062 segment.go:SetEnabled:560 → Segment: Azfunc
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [*.lua *.rockspec]
[TRACE] 06:55:28.062 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → folders: [lua]
[DEBUG] 06:55:28.062 properties.go:GetStringArray:52 → extensions: [host.json local.settings.json function.json]
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[TRACE] 06:55:28.063 shell.go:Flags() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Kotlin
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[TRACE] 06:55:28.063 shell.go:Flags() - 0s
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Kubectl
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Nx
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → parse_kubeconfig: false
[TRACE] 06:55:28.063 shell.go:Flags() - 0s
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → extensions: [workspace.json nx.json]
[DEBUG] 06:55:28.063 shell.go:Getenv:326 → C:\Users\user\AppData\Local
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[TRACE] 06:55:28.063 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:Flags() - 0s
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → extensions: [*.kt *.kts *.ktm]
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.063 properties.go:GetString:28 → files
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Cf
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → extensions: [manifest.yml mta.yaml]
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Crystal
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → extensions: [*.cr shard.yml]
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.063 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 segment.go:SetEnabled:560 → Segment: Node
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 06:55:28.063 properties.go:GetStringArray:52 → folders: []
[DEBUG] 06:55:28.063 properties.go:GetBool:22 → home_enabled: false
[TRACE] 06:55:28.063 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.063 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.063 shell.go:HasFilesInDir:409 → false
[TRACE] 06:55:28.063 shell.go:HasFilesInDir(azureProfile.json) - 914.4µs
[DEBUG] 06:55:28.063 shell.go:HasFilesInDir:409 → false
[TRACE] 06:55:28.063 shell.go:HasFilesInDir(azureProfile.json) - 0s
[DEBUG] 06:55:28.063 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.063 shell.go:Getenv(POSH_AZURE_SUBSCRIPTION) - 0s
[ERROR] 06:55:28.092 shell.go:CommandPath:557 → The system cannot find the file specified.
[TRACE] 06:55:28.092 shell.go:CommandPath(cf) - 29.9211ms
[TRACE] 06:55:28.092 shell.go:HasCommand(cf) - 29.9211ms
[ERROR] 06:55:28.093 shell.go:CommandPath:557 → The system cannot find the file specified.
[TRACE] 06:55:28.093 shell.go:CommandPath(kubectl) - 30.1419ms
[TRACE] 06:55:28.093 shell.go:HasCommand(kubectl) - 30.1419ms
[DEBUG] 06:55:28.093 text.go:Render:72 → Rendering template:  {{ round .PhysicalPercentUsed .Precision }}% 
[DEBUG] 06:55:28.093 text.go:Render:72 → Rendering template:   {{ .Name }} 
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template:  {{ .CurrentDate | date .Format }}
[TRACE] 06:55:28.094 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.094 shell.go:TemplateCache() - 0s
[TRACE] 06:55:28.094 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.094 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.095 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.095 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.095 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.095 shell_windows.go:TerminalWidth:111 → terminal width: 280
[TRACE] 06:55:28.095 shell_windows.go:TerminalWidth() - 0s
[TRACE] 06:55:28.095 shell.go:Shell() - 0s
[TRACE] 06:55:28.095 shell.go:Shell() - 0s
[TRACE] 06:55:28.095 shell.go:Shell() - 0s
[TRACE] 06:55:28.095 shell.go:Shell() - 0s
[TRACE] 06:55:28.095 shell.go:Flags() - 0s
[TRACE] 06:55:28.095 shell.go:Flags() - 0s
[TRACE] 06:55:28.095 shell.go:Pwd() - 0s
[TRACE] 06:55:28.095 shell.go:Pwd() - 0s
[TRACE] 06:55:28.095 shell.go:Flags() - 0s
[DEBUG] 06:55:28.095 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.095 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.095 segment.go:SetEnabled:560 → Segment: Status
[TRACE] 06:55:28.095 shell.go:StatusCodes() - 0s
[TRACE] 06:55:28.095 shell.go:Pwd() - 0s
[DEBUG] 06:55:28.095 text.go:Render:72 → Rendering template: {{ .Code }}
[DEBUG] 06:55:28.095 shell_windows.go:58 → C:\Users\user
[TRACE] 06:55:28.096 shell.go:GOOS() - 0s
[DEBUG] 06:55:28.096 shell_windows.go:58 → C:\Users\user
[DEBUG] 06:55:28.096 segment.go:SetEnabled:560 → Segment: Text
[TRACE] 06:55:28.096 shell.go:GOOS() - 0s
[TRACE] 06:55:28.096 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.096 segment.go:SetEnabled:560 → Segment: Spotify
[TRACE] 06:55:28.096 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.096 properties.go:GetBool:22 → always_enabled: true
[TRACE] 06:55:28.096 shell.go:TemplateCache() - 0s
[ERROR] 06:55:28.160 shell_windows.go:QueryWindowTitles:75 ↓
    no matching window title found
    
[TRACE] 06:55:28.160 shell_windows.go:QueryWindowTitles(^(Spotify.*)|(.*\s-\s.*)$) - 64.2112ms
[TRACE] 06:55:28.221 shell_windows.go:QueryWindowTitles(^(Spotify.*)) - 60.6478ms
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: ╰─
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: 
[DEBUG] 06:55:28.222 shell.go:Getenv:326 → NO DATA
[TRACE] 06:55:28.222 shell.go:Getenv(TERM_PROGRAM) - 0s
[TRACE] 06:55:28.222 shell.go:Shell() - 0s
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: diamond
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: plain
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: {{ if gt .Code 0 }}#ef5350{{ end }}
[TRACE] 06:55:28.222 shell.go:TemplateCache() - 0s
[DEBUG] 06:55:28.222 text.go:Render:72 → Rendering template: plain
[TRACE] 06:55:28.222 shell.go:Shell() - 0s
[TRACE] 06:55:28.222 shell.go:Shell() - 0s
[TRACE] 06:55:28.222 shell.go:Flags() - 0s
[TRACE] 06:55:28.222 shell.go:Shell() - 0s
[TRACE] 06:55:28.222 shell.go:Shell() - 0s
[DEBUG] 06:55:28.222 shell.go:Getenv:326 → C:\Users\user\AppData\Local
[TRACE] 06:55:28.222 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 06:55:28.222 shell_windows.go:CachePath() - 0s
[TRACE] 06:55:28.222 shell.go:Flags() - 0s
```

--- a/src/shell/scripts/omp.ps1
+++ b/src/shell/scripts/omp.ps1
@@ -359,7 +359,8 @@ Example:
         # for details about the trick to detect a debugging context, see these comments:
         # 1) https://github.com/JanDeDobbeleer/oh-my-posh/issues/2483#issuecomment-1175761456
         # 2) https://github.com/JanDeDobbeleer/oh-my-posh/issues/2502#issuecomment-1179968052
-        if ($null -ne $PSDebugContext -or -not ((Get-PSCallStack).Location -join "").StartsWith("<")) {
+        # 3) https://github.com/JanDeDobbeleer/oh-my-posh/issues/5153
+        if ($Host.Runspace.Debugger.InBreakpoint) {
             $script:PromptType = "debug"
             return
         }

